documentation/schema.json

```diff
@@ -1183,6 +1183,13 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "DefaultSharedPath": {
+          "type": [
+            "null",
+            "string"
+          ],
+          "description": "The default path where assets (dumps, socket files, libraries) will be shared between dotnet-monitor and target processes."
```

Comment:
I think we should expand this documentation. For example, we should mention what settings are superseded and that we switch to listen mode automatically.

Reply:
I'm not sure how to describe all of that in this description without writing a book. How's this?

The default path where assets will be shared between dotnet-monitor and target processes. Dumps are temporarily stored under this path or in a sub folder unless DumpTempFolder is specified. Shared libraries are stored under this path or in a sub folder unless SharedLibraryPath is specified. On non-Windows, dotnet-monitor runs in listen mode with a Unix domain socket named 'dotnet-monitor.sock' immediately under this path unless the diagnostic port is specified on the command line or the DiagnosticPort options are specified.